### ISSUE TITLE
Show percentile in rank_icon=percentile

### ISSUE THREAD

**Issue by Madhav-MKNC** [anuraghazra/github-readme-stats#2867]
### Is your feature request related to a problem? Please describe.

Currently, the GitHub stats card displays the percentile in the form of the "in the top X percentage". 

### Describe the solution you'd like

It would be better if it could also display the percentile in the form "Y percentile" where Y = (100-X).

**Comment by rickstaa** [comment]
@Madhav-MKNC, thanks for your feature request. We briefly merged what you want in https://github.com/anuraghazra/github-readme-stats/pull/2858, but we reverted it since the percentile was more apparent. Can you checkout if this is what you had in mind:


```md
![Anurag's GitHub stats](https://github-readme-stats-git-addprogressrankicon-rickstaa.vercel.app/api?username=anuraghazra\&rank_icon=progress)
```

![Anurag's GitHub stats](https://github-readme-stats-git-addprogressrankicon-rickstaa.vercel.app/api?username=anuraghazra\&rank_icon=progress)

**Comment by Madhav-MKNC** [comment]
Yes, It's exactly what I was asking for, thanks for sharing.

**Comment by Madhav-MKNC** [comment]
Can we have a separate card for just the rank_icon, I think many users like me like to show all of followings on their profile: grade, percentile, progress

**Comment by rickstaa** [comment]
> Yes, It's exactly what I was asking for, thanks for sharing.

Thanks! @qwerty541, what are your thoughts about adding the progress code back in the master branch?

**Comment by qwerty541** [comment]
> > Yes, It's exactly what I was asking for, thanks for sharing.
> 
> Thanks! @qwerty541, what are your thoughts about adding the progress code back in the master branch?

As i wrote in https://github.com/anuraghazra/github-readme-stats/pull/2859#pullrequestreview-1487346398 i think that it is impossible to understand what the `100 - percentile` (i.e. 80%) means for random profile visitor. Because of this, i think that it is better to not return this functionality to master branch. But we can keep this issue open and if it receive a lot of community support (#1935) may be we can still add it later.

@Madhav-MKNC you are able to deploy GRS at this point https://github.com/anuraghazra/github-readme-stats/tree/a362af0ebc7bbcd6821e70ba559fecde4ce3bf2e on your own vercel instance if you need this right now. If you need something for own use it does not means that it should be publicly deployed since it may turn out that no one needs it except you.

**Comment by rickstaa** [comment]
> > > Yes, It's exactly what I was asking for, thanks for sharing.
> > 
> > 
> > Thanks! @qwerty541, what are your thoughts about adding the progress code back in the master branch?
> 
> As i wrote in [#2859 (review)](https://github.com/anuraghazra/github-readme-stats/pull/2859#pullrequestreview-1487346398) i think that it is impossible to understand what the `100 - percentile` (i.e. 80%) means for random profile visitor. Because of this, i think that it is better to not return this functionality to master branch. But we can keep this issue open and if it receive a lot of community support (#1935) may be we can still add it later.
> 
> @Madhav-MKNC you are able to deploy GRS at this point https://github.com/anuraghazra/github-readme-stats/tree/a362af0ebc7bbcd6821e70ba559fecde4ce3bf2e on your own vercel instance if you need this right now. If you need something for own use it does not means that it should be publicly deployed since it may turn out that no one needs it except you.

@Madhav-MKNC I have to agree with @qwerty541 that since the stat is not apparent, it is better to not merge it until there is more demand. Let's see if this feature request gets enough 👍🏻 to merge it in the future. Neverteless, I created https://github.com/anuraghazra/github-readme-stats/pull/2875, which you can use on your own Vercel instance if you already want to use this feature.
